.eslintrc

```diff
@@ -0,0 +1,31 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
```

Comment:
Could you take a quick look at few of the presets (eg: Airbnb, Google, Facebook, etc) and suggest which one would work the best for us?